helmfile.d/charts/log-manager/values.yaml

```diff
@@ -3,15 +3,22 @@ nameOverride: ""
 
 imagePullSecrets: []
 image:
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
   repository: ghcr.io/elastisys/compliantkubernetes-apps-log-manager
-  tag: ""
+  tag: "0.3.0"
```

Comment:
Why always? And the tag should be set in Chart.yaml as the app version.

Reply:
Was there for testing purpose will remove.